### PR TITLE
Add proposal for lvcreate options

### PR DESCRIPTION
**Pull request by lentzi90**
LVcreate options is a feature to allow passing options to `lvcreate`
when using TopoLVM.
For example, set the `alloc` flag or specify a `raid` configuration.
This proposal suggests adding a field `lvcreate-options` that would be
used to pass options directly to `lvcreate`.

This replaces part of #416.

### CODE REVIEW

**Comment by llamerada-jp** [comment]
Hi @lentzi90,

Thank you for your new proposal. We think Option A is better than B. The reason for this is like below.

- Fewer changes are required.
- Some options of lvcreate are depending on the structures of VG (like `--mirrors 2` you pointed). In option A, options for lvcreate and VG correspond 1:1. It looks easy to understand for administrators and is good to avoid problems.

About `stripe` and `stripe-size options`, We don't think we have to depreciate. I think it is better to annotate for the document of `lvcreate-options`.
About scheduling and multiple device-classes, We agree with your proposal. This means that no changes for these.

**Comment by lentzi90** [comment]
@llamerada-jp great!
How should I move forward here? Do I need to make any changes to the proposal? Or should I wait for more reviews first?

**Comment by llamerada-jp** [comment]
@lentzi90 
The other members have checked this proposal. And they agreed above comment. Would you please fill the `TODO`? I'll merge this PR after filling it. So, you can start implementing it.

**Review by toshipp — APPROVED** [review]
LGTM

**Review by llamerada-jp — APPROVED** [review]
LGTM

**Pull request merged**
